lib/recorder.js

```diff
@@ -233,7 +233,7 @@ function record(rec_options) {
   const logging = (optionsIsObject && rec_options.logging) || console.log
   let use_separator = true
   if (optionsIsObject && _.has(rec_options, 'use_separator')) {
-    use_separator = rec_options.use_separator
+    ;({ use_separator } = rec_options)
```

Comment:
this reads very odd ... 🤔 any alternative idea? Could the enforced destructing be disabled for lines where existing variables are set?

Reply:
Yea, I agree… I removed it the first time around, but now lint is complaining.

It looks like it can be turned off for assignment expressions: https://eslint.org/docs/rules/prefer-destructuring

I'll give that a shot.